autogpt/memory_management/summary_memory.py

```diff
@@ -7,115 +7,115 @@
 
 cfg = Config()
 
-
-def get_newly_trimmed_messages(
-    full_message_history: List[Dict[str, str]],
-    current_context: List[Dict[str, str]],
-    last_memory_index: int,
-) -> Tuple[List[Dict[str, str]], int]:
+class SummaryMemoryMixin:
+    def get_newly_trimmed_messages(
+        self,
+        current_context: List[Dict[str, str]],
+        last_memory_index: int,
+    ) -> List[Dict[str, str]]:
```

Comment:
now it's returning only one thing.
the other thing it was returning is just updated as an attribute of the agent class, thanks to the mixin.